releasenotes/notes/55281.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: traffic-management
+releaseNotes:
+  - |
+    **Improved** iptables binary detection to verify a degree of baseline kernel support exists,
```

Comment:
No action required. I understand this is a cherry-pick, but this sentence could be a bit misleading to the users. Since `iptables-nft` is a compatibility layer that uses iptables syntax, whereas `nft` is a separate CLI binary that supports `native nftables` syntax to leverage its full feature set, this might imply to users that Istio supports `native nftables` via the `nft` binary (if present on the host/container).